### ISSUE TITLE
Error when parsing unusual spaced operators in Control.Lens.Combinators

### ISSUE THREAD

**Issue by meditans** [haskell-suite/haskell-src-exts#322]
I was trying to parse the source of
[Control.Lens.Combinators](https://hackage.haskell.org/package/lens-4.14/docs/src/Control.Lens.Combinators.html), and I encountered a problem in the line beginning with

      , ( #~ )

I think the error is due to the unusual spacing of the operator.

Using the function `parseFileContentWithComments` with

``` haskell
parseMode = defaultParseMode { fixities = Nothing
                             , extensions = defaultExtensions
                             }
```

where defaultExtensions is the union of all "reasonable" extensions (like it's
done by hlint), I get the error:

```
ParseFailed (SrcLoc "<unknown>.hs" 95 7) "Parse error: #"
```

Do you have any idea on what the problem could be, @mpickering? Is this related to fixities?

**Comment by mpickering** [comment]
Which version of HSE are you using? You should make sure to try first with 1.18.2, if that fails to work then please comment again.

**Comment by meditans** [comment]
Yeah, sorry I forgot to mention, I did already upgrade to 1.18.2 when I opened the issue.

**Comment by mpickering** [comment]
I don't know what the problem is. You should try disabling the `OverloadedLabels` extension.
